Doc/library/inspect.rst

```diff
@@ -214,6 +227,18 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 |                   | arguments and local       |
 |                 |                   | variables                 |
 +-----------------+-------------------+---------------------------+
+|                 | co_lines          | returns an iterator that  |
+|                 |                   | yields successive         |
+|                 |                   | bytecode ranges           |
++-----------------+-------------------+---------------------------+
+|                 | co_positions      | returns an iterator of    |
+|                 |                   | source code positions for |
+|                 |                   | each bytecode instruction |
++-----------------+-------------------+---------------------------+
+|                 | replace           | returns a copy of the     |
+|                 |                   | code object with a new    |
+|                 |                   | values                    |
```

Comment:
"with a new values"? is that correct in English? I'm asking but I'm not a native English speaker.

Reply:
I'm also not a native English speaker :(
I think the article 'a' is unnecessary, but it needs confirmation from a native speaker.

Reply:
I would say: "with new values".